tests/container/BUILD

```diff
@@ -378,7 +378,7 @@ py_test(
 # BEGIN_DO_NOT_IMPORT
 new_container_push(
     name = "new_push_test_skip_unchanged_digest_unchanged_tag_1",
-    format = "docker",
+    format = "Docker",
```

Comment:
cant we just remove this since its the default?

Reply:
Instead, I made the attribute mandatory. My goal is to make `new_container_push` a drop-in replacement for `container_push` and this attribute is mandatory in `container_push`.